package.json

```diff
@@ -20,7 +20,7 @@
     "test": "test"
   },
   "scripts": {
-    "tsc": "rm -rf build/*/** && tsc",
+    "tsc": "rm -rf build && tsc",
```

Comment:
I actually don't want to remove files directly under `build/`; I only wanna remove files in `build/`'s subfolders.  

Two reasons:
1) when I run `test`, I shouldn't have to wipe out the bundled files and the schema (i don't rebuild them anyway.) 
2) These files at the top-level of `build/` should never get outdated unless we do major folder restructure (unlike ones in src/ and test/) so it's okay just to remove subfolder like this.